### ISSUE TITLE
'list' object has no attribute 'get_body' in EventHub trigger

### ISSUE THREAD

**Issue by Hazhzeng** [Azure/azure-functions-python-worker#699]
### TL;DR
Change the **cardinality** to **one** in EventHub binding **function.json**.

### Background
Recently, we upgraded the function host version to 2.0.13907 for v2 and 3.0.13901 for v3. We also upgraded the Python worker from 1.0.3 to 1.1.1, and the change has been deployed to production on June 21, 2020.

In Python worker release [1.0.4](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.0.4), we fix the function.json **"cardinality"="many"** handling for EventHub triggers.

When setting **"cardinality"="many"**, the Python handler signature should accept a list of events instead of a single one.
```python
from typing import List
import azure.functions as func

def main(events: List[func.EventHubEvent]):
```

### Affects on Python Functions Applications
Some EventHub trigger may have **"cardinality"="many"** setting in their **function.json** file, and if the EventHub trigger only handles a single event, this will **BREAK**. 
```python
import azure.functions as func

def main(event: func.EventHubEvent):
```
Since the event parameter now has a proper list of events, instead of a single event.

### How To Mitigate
In EventHub triggers **function.json**, change the **cardinality** to **one**. If you intend to update the **main() function signature** in your EventHub trigger, please also update the **name** in **function.json**.

Alternatively, you can keep **cardinality=many** and change the EventHub trigger to process multiple events in one go. Remember to change the **name** in **function.json** into **events**. 
from
```python
def main(event: func.EventHubEvent):
    event.get_body()
```
to
```python
def main(events: List[func.EventHubEvent]):
    for event in events:
        event.get_body()
```

Sorry for the inconvenience, if you have more questions related to this issue, please let me know and comment below.

**Comment by Sniedes722** [comment]
@Hazhzeng it should be added that this will also break your existing `function.json` setup as the previous implies you have a setting in that file: `"name": "event"` which should be changed to `"name":"events"`

**Comment by Hazhzeng** [comment]
If you're facing similar issue in ServiceBus, please check this announcement.
https://github.com/Azure/azure-functions-python-worker/issues/783
